client/tx/search.go

```diff
@@ -120,6 +134,28 @@ func searchTxs(cliCtx context.CLIContext, cdc *codec.Codec, tags []string) ([]In
 	return info, nil
 }
 
+func getKeyFromBechPrefix(key string, value string) (string, string, string) {
+	if strings.HasSuffix(key, "_bech32") {
+		prefix := strings.Split(value, "1")[0]
+		bz, err := sdk.GetFromBech32(value, prefix)
+		if err != nil {
+			return "", "", err.Error()
+		}
+
+		switch prefix {
+		case sdk.Bech32PrefixAccAddr:
+			value = sdk.AccAddress(bz).String()
+		case sdk.Bech32PrefixValAddr:
+			value = sdk.ValAddress(bz).String()
```

Comment:
How does `value` here differ from the `value` passed in? Aren't both Bech32-prefixed strings?

Reply:
I think this logic comes from when we didn't had `bech32` prefixes (i.e hex addresses). Should I just delete this function then ?

Reply:
Ahhh, in that case, yes!

Reply:
works without this function, so I'll just delete it